v2/src/components/appInfoForm/style.css

```diff
@@ -157,15 +157,22 @@
 }
 
 .app-info-form-field-input {
-  border: 1px solid rgb(72, 72, 72);
+  border: 1px solid rgb(69, 69, 69);
+  background-color: rgb(0, 0, 0);
+  color: rgb(255, 255, 255);
   box-shadow: 0px 0px 0px 0px transparent;
   outline: none;
   transition: border .15s, box-shadow .15s;
+  padding: 8px 20px;
+  font-family: Roboto;
```

Comment:
font family according to this [chat ](https://supertokens.slack.com/archives/C03HY1RQVB2/p1661862415953629?thread_ts=1661852504.367859&cid=C03HY1RQVB2)